python-spec/src/somacore/query/query.py

```diff
@@ -241,12 +241,28 @@ def varp(self, layer: str) -> data.SparseRead:
         """
         return self._axisp_inner(_Axis.VAR, layer)
 
+    def obsm(self, layer: str) -> data.SparseRead:
+        """Returns an ``obsm`` layer as a sparse read.
+        Lifecycle: maturing
```

Comment:
I think these should be `experimental` until they have been in a release or two.  They can't be maturing if they haven't even been merged :-)